### PR TITLE
ROC: 2217 enter claim evidence

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-2217

### Change description
Evidence page for claimant

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [x] New pages included in a11y tests
- [x] Task list and task completeness checks updated
- [x] Check and send page updated
- [ ] ~Routes, page content and page flows of new features are toggled~
- [x] UI changes look good on mobile
- [ ] ~Required Google Analytics events are being sent~

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/response/routes/evidence.ts [hunk 1]
```diff
@@ -20,8 +19,7 @@ function renderView (form: Form<Evidence>, res: express.Response): void {
 
   res.render(page.associatedView, {
     form: form,
-    claimantName: claim.claimData.claimant.name,
-    allEvidenceTypes: EvidenceType.all()
+    claimantName: claim.claimData.claimant.name
```
Comment (robertparkinson) [inline src/main/features/response/routes/evidence.ts]: claim.claimData.claimant.name can be accessed directly on the view { claimantName: claim.claimData.claimant.name }

File: src/main/features/claim/views/check-and-send.njk [hunk 2]
```diff
@@ -74,6 +74,13 @@
     {{ row(timelineRow.date, timelineRow.description, '', loop.index0 === lastItemIndex) }}
     {% endfor %}
 
+    {{ row('Your evidence (optional):', '', paths.evidencePage.uri, false ) }}
+    {% set allEvidenceRows = draftClaim.evidence.getPopulatedRowsOnly() %}
+    {% set lastItemIndex = allEvidenceRows.length - 1 %}
+    {% for evidenceRow in allEvidenceRows %}
+    {{ row(evidenceRow.type.displayValue, evidenceRow.description | default(''), '', loop.index0 === lastItemIndex) }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: mind intending this please?

File: src/main/features/response/routes/evidence.ts (hunk 1 above)
Comment (yarpo) [reply src/main/features/response/routes/evidence.ts]: It's going to be removed in next iteration as information about claimant name is not displayed anymore in the current version of prototype.

I'd leave it for 2-3 days as it is.

File: src/main/features/claim/views/check-and-send.njk (hunk 2 above)
Comment (yarpo) [reply src/main/features/claim/views/check-and-send.njk]: ok

**Comment by pragnesh-patel** [comment]
It has been tested with QA
The backend is complete and deployed to production
Checkpoint of reviewing test with QA done.

**Comment by pragnesh-patel** [comment]
Checkpoint of checking AAT by QA - To be Done

**Pull request merged**